exec/java-exec/src/test/java/org/apache/drill/test/ClusterFixture.java

```diff
@@ -203,7 +219,15 @@ private void createConfig(FixtureBuilder builder) throws Exception {
     if (builder.configResource != null) {
       config = DrillConfig.create(builder.configResource);
     } else if (builder.configProps != null) {
-      config = DrillConfig.create(configProperties(builder.configProps));
+      // Create Config object out of all the String type configuration.
```

Comment:
Suggestion to keep things tidy. Keep all properties (simple and complex) in the builder's `configProps` object. Then, in `configProperties`, do the mapping as required to create the `DrillConfig`. If a value is a collection, use the new code to set the value. Else, convert the value using `toString()`.